packages/cli-helpers/src/lib/installHelpers.ts

```diff
@@ -0,0 +1,24 @@
+import execa from 'execa'
+
+export const addWebPackages = (webPackages: string[]) => ({
+  title: 'Adding required web packages...',
+  task: async () => {
+    const args = ['workspace', 'web', 'add', ...webPackages]
+    await execa('yarn', args)
+  },
+})
+
+export const addApiPackages = (apiPackages: string[]) =>
+  apiPackages.length > 0 && {
+    title: 'Adding required api packages...',
+    task: async () => {
+      await execa('yarn', ['workspace', 'api', 'add', ...apiPackages])
+    },
+  }
```

Comment:
This one's changed a bit; the deleted code is up to date but not this one

```suggestion
export const addApiPackages = (apiPackages: string[]) => ({
  title: 'Adding required api packages...',
  task: async () => {
    await execa('yarn', ['workspace', 'api', 'add', ...apiPackages])
})
```